embedded-hal-bus/Cargo.toml

```diff
@@ -16,7 +16,7 @@ version = "0.1.0-rc.1"
 [features]
 std = []
 async = ["dep:embedded-hal-async"]
-defmt-03 = ["dep:defmt-03", "embedded-hal/defmt-03", "embedded-hal-async?/defmt-03"]
+defmt-03 = ["dep:defmt-03", "embedded-hal/defmt-03", "dep:embedded-hal-async", "embedded-hal-async?/defmt-03"]
```

Comment:
enabling `defmt-03` must not enable `embedded-hal-async`. It should enable the `embedded-hal-async/defmt-03` feature only if `embedded-hal-async`was already enabled by the `async` feature.

Reply:
oops, good catch, thx!  I have seen some incorrect `?` usage before, so jumped to conclusion too fast.